web/src/compose_closed_ui.ts

```diff
@@ -114,20 +129,24 @@ function toggle_direct_message_button_visibility(is_direct_message_narrow) {
     }
 }
 
-function update_buttons(text_stream, is_direct_message_narrow, disable_reply) {
+function update_buttons(
+    text_stream: string,
+    is_direct_message_narrow?: boolean,
+    disable_reply?: boolean,
+): void {
     const text_conversation = $t({defaultMessage: "New direct message"});
     update_new_conversation_button(text_stream, is_direct_message_narrow);
     update_new_direct_message_button(text_conversation);
     update_reply_button_state(disable_reply);
     toggle_direct_message_button_visibility(is_direct_message_narrow);
 }
 
-export function update_buttons_for_private() {
+export function update_buttons_for_private(): void {
     const text_stream = $t({defaultMessage: "Start new conversation"});
     const is_direct_message_narrow = true;
     if (
         !narrow_state.pm_ids_string() ||
-        people.user_can_direct_message(narrow_state.pm_ids_string())
+        people.user_can_direct_message(narrow_state.pm_ids_string()!)
```

Comment:
You can avoid the unchecked assertion here by just setting a local variable to `const pm_ids_string = narrow_state.pm_ids_string());` and working with that -- then the immediately previous check will exclude that possibility for you.

Reply:
Good catch, changed that.